test/spec/geo/geometry-views/shared-tests-for-path-views.js

```diff
@@ -161,12 +161,17 @@ module.exports = function (Path, MapView, PathView) {
         mapView: this.mapView
       });
 
-      this.geometryView.render();
+      // Listen for the map to be loaded
+      this.mapView.listenOnce('idle', function () {
```

Comment:
Not very elegant but perhaps we can check if `this.mapView` is an instance of `src/geo/gmaps/gmaps-map-view.js` and wait to the `idle` event only in that case? So that we can get rid of `listenOnce` which was only added for testing purposes...